apps/www/components/Access/Grants.js

```diff
@@ -17,11 +17,15 @@ const dayFormat = timeFormat('%e. %B %Y')
 const AccessGrants = ({ accessGrants, inNativeIOSApp, t }) => {
   const maxEndAt =
     accessGrants.length > 0 &&
-    accessGrants.reduce(
-      (acc, grant) =>
-        new Date(grant.endAt) > acc ? new Date(grant.endAt) : acc,
-      new Date(),
-    )
+    accessGrants
+      .filter((grant) => {
+        return grant.campaign.type === 'REGULAR'
+      })
+      .reduce(
+        (acc, grant) =>
+          new Date(grant.endAt) > acc ? new Date(grant.endAt) : acc,
+        null,
```

Comment:
Why did you get rid of the starting value of the accumulator?
I think this won't work as it used to with this chance. I am unsure if Date > null will return true, if it doesn't the accumulator will never be changed, and therefor it would reduce to null.